### DISCUSSION TITLE
Example of image upload via API

### DISCUSSION THREAD

**Discussion by icylian** [photoprism/photoprism#4493] [Q&A]
Hello, I attempted to upload images to my own Photoprism instance using the official API. When I used Postman to send a POST form-data request to https://myserver/api/v1/users/{uid}/upload/{token}, the interface returned a 200 OK status, but it showed that 0 files were uploaded. I would like to know where the issue might have occurred in the above steps. If there is a successful upload example, I would appreciate it if you could provide it for reference.

Additionally, I noticed that the official documentation [here](https://docs.photoprism.dev/#/Files/post_users__uid__upload__token_) mentions that the upload endpoint is /users/{uid}/upload/{token}, but it seems they omitted the preceding /api/v1. When I tried this endpoint, Photoprism returned a 404 error.

**Comment by lastzero** [comment]
You need to import the files after uploading them, just like our web UI does.
